library/hatomap.py

```diff
@@ -0,0 +1,413 @@
+from __future__ import annotations
+
+import math
+import random
+import string
+from dataclasses import dataclass, field
+from multiprocessing import Pool
+
+from typing import Any, List, Optional, Tuple, Union
+
+import cv2
```

Comment:
https://stackoverflow.com/questions/50612169/pylint-not-recognizing-cv2-members

```suggestion
from cv2 import cv2
```

こうしたら良い説？